Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -1854,6 +1859,29 @@ extension TelemetryWrapper {
                 let properties = GleanMetrics.AppErrors.LargeFileWriteExtra(size: quantity)
                 GleanMetrics.AppErrors.largeFileWrite.record(properties)
             }
+        case(.action, .tap, .fxSuggest, _, let extras ):
+            guard let contextId = UUID(uuidString: TelemetryContextualIdentifier.contextId ?? "") else { return }
+            if let interactionInfo = extras?[EventValue.fxSuggestionClickInfo.rawValue] as? RustFirefoxSuggestionInteractionInfo {
+                switch interactionInfo {
+                    case .amp(blockId: let blockId, advertiser: let advertiser, iabCategory: let iabCategory, reportingURL: let reportingURL):
+                        GleanMetrics.FxSuggest.contextId.set(contextId)
+                        GleanMetrics.FxSuggest.pingType.set("fxsuggest-click")
+                        GleanMetrics.FxSuggest.blockId.set(blockId)
+                        GleanMetrics.FxSuggest.advertiser.set(advertiser)
+                        GleanMetrics.FxSuggest.reportingUrl.set(url: reportingURL)
+                        GleanMetrics.FxSuggest.iabCategory.set(iabCategory)
+                        if let position = extras?[EventValue.fxSuggestionPosition.rawValue] as? Int64 {
+                            GleanMetrics.FxSuggest.position.set(position)
+                        }
+                    case .wikipedia:
+                        GleanMetrics.FxSuggest.pingType.set("fxsuggest-click")
+                        GleanMetrics.FxSuggest.contextId.set(contextId)
```

Comment:
```suggestion
                        GleanMetrics.FxSuggest.contextId.set(contextId)
                        GleanMetrics.FxSuggest.advertiser.set("wikipedia")
```

(We do the same on [Android](https://github.com/mozilla-mobile/firefox-android/blob/8bb2f17dd063c8de177cefcff18c86002665c9be/fenix/app/src/main/java/org/mozilla/fenix/components/metrics/MetricController.kt#L300)).

Reply:
right, will do